docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,245 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
+The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
+
+This process works with local VMs, physical servers and/or cloud servers.
+It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
+
+**The `kubeadm` tool is currently in alpha but please try it out and give us [feedback](/docs/getting-started-guides/kubeadm/#feedback)!**
+
+## Prerequisites
+
+1. One or more machines running Ubuntu 16.04 or CentOS 7
+1. 2GB or more of RAM per machine
```

Comment:
Why is this here?
I'm running Kubernetes fine on my 1GB Raspberry Pi's, this only confuses users, as 2GB is not a minimum
I've also run Kubernetes on 512 MB droplets

Please remove